dev/breeze/doc/BREEZE.md

```diff
@@ -32,6 +32,8 @@
         alt="Airflow Breeze - Development and Test Environment for Apache Airflow">
 </div>
 
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
Copy * Paste mistake :) ?

Reply:
wow, why I could be having conflicts from my last PR that already was merged (Breeze for windows documentation)? 🙄

Reply:
Happens sometimes, when there som whitespace differences etc. I tend to get similar artifacts from time to time - repeated lines. Usually there some results of resolving conflicts slightly differently in two branches.